### DISCUSSION TITLE
Locally hosting Redirector

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#463] [(D) Developer]
While we haven't updated instruction docs for Redirector, would anybody be willing to setup alternative Redirector links?

https://github.com/Prodigy-Hacking/Redirector#locally-hosting-redirector-advanced

**Comment by ArcerionDev** [comment]
you could use https://www.000webhost.com/

**Reply by PatheticMustan** [reply]
> you could use https://www.000webhost.com/

no

**Comment by ArcerionDev** [comment]
and freenom

**Comment by poIarbear** [comment]
bro arc my discord got hacked heres my new one sun#7561
hope you see this :c
